components/brave_wallet_ui/utils/string-utils.ts

```diff
@@ -6,7 +6,19 @@ export const toProperCase = (value: string) =>
     (txt) => txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase())
 
 export const isRemoteImageURL = (url?: string) =>
-  url?.startsWith('http://') || url?.startsWith('https://') || url?.startsWith('data:image/')
+  url?.startsWith('http://') || url?.startsWith('https://') || url?.startsWith('data:image/') || url?.startsWith('ipfs://')
 
 export const isValidIconExtension = (url?: string) =>
-  url?.endsWith('.jpg') || url?.endsWith('.jpeg') || url?.endsWith('.png') || url?.endsWith('.svg')
+  url?.endsWith('.jpg') || url?.endsWith('.jpeg') || url?.endsWith('.png') || url?.endsWith('.svg') || url?.endsWith('.gif')
```

Comment:
Are we removing any parameter from the URL?
What happens when the filename matches the allowlist but the infered mime-type is different (e.g. If I rename an SVG to PNG); isn't the picture loaded anyway even if mislabeled?

Reply:
@thypon We are not removing any parameters from the URL.
If the inferred mime-type is different and the image passes the allowlist it will be displayed. No mime-type checks are performed at the moment.

Reply:
Are we storing and serving any NFT from a `chrome://` or `chrome-untrusted://`?

Reply:
@thypon No, all NFT images are using `chrome-untrusted`